### PR TITLE
Update OpenRelay information

### PR DESCRIPTION
**Pull request by AusIV**

### CODE REVIEW

**Comment by tomhschmidt** [comment]
Hey @AusIV -- is OpenRelay only on mainnnet? If so, can you remove the Ropsten / 3 entry in the networks array? That must have been my mistake.

File: relayers.json [hunk 1]
```diff
@@ -39,7 +39,15 @@
...
+        "logo": "openrelay.png",
         "networks": [
+            {
+                "networkId": 1,
+                "sra_http_endpoint": "https://api.openrelay.xyz/v0/",
+                "static_order_fields": {
+                    "fee_recipient_addresses": ["0xc22d5b2951db72b44cfb8089bb8cd374a3c354ea"]
+                }
+            },
             {
```
Comment (tomhschmidt) [inline relayers.json]: Does OpenRelay not have a Ropsten network orderbook? If not, can you remove the Ropsten (network 3) entry in this array as well?

**Comment by AusIV** [comment]
OpenRelay is on both mainnet and ropsten through the same endpoint. We use the exchange contract address to distinguish which network orders need to be validated against.

**Comment by tomhschmidt** [comment]
Gotcha. Thanks!

**Pull request merged**